packages/cascara/src/private/ViewConfig/ViewConfigItem.js

```diff
@@ -77,7 +77,6 @@ const ViewConfigItem = ({
   return (
     <MenuItem
       {...state}
-      {...rest}
```

Comment:
I guess technically it really is this simple, huh? We can probably take out the extra memo on the object, but for now I think this is sufficient. :)

Reply:
Brian, I tried that, but I got an error message from eslint, that's why I left it.